doc/api/n-api.md

```diff
@@ -4188,6 +4234,75 @@ with the resulting JavaScript constructor (which is returned in the `result`
 parameter) and freed whenever the class is garbage-collected by passing both
 the JavaScript function and the data to [`napi_add_finalizer`][].
 
+### napi_define_class_with_length
```

Comment:
I'd wonder if there are options to merge `napi_define_class` and `napi_define_class_with_length` in documents since they are very similar in shape and effects, except for the parameter `arguments_length`. For most of the content of the doc are identical, these might not be easy to spot out the difference.